crates/undo/src/lib.rs

```diff
@@ -331,6 +301,7 @@ impl EditorChange for RemovedEntity {
                 let id = world
                     .spawn_empty()
                     .insert(OneFrameUndoIgnore::default())
+                    .insert(UndoMarker)
```

Comment:
This can be replaced more effectively by:
`.insert((OneFrameUndoIgnore::default(),UndoMarker))`

Avoid doing multiple insert calls as they downfall performance

Reply:
Fixed